tests/integration/utils.go

```diff
@@ -772,15 +793,37 @@ func startNodes(s *state, action Start) {
 			addresses = append(addresses, addr.String())
 		}
 		opts = append(opts, netConfig.WithListenAddresses(addresses...))
-		node, err := setupNode(s, opts...)
-		require.NoError(s.t, err)
+		node, err := setupNode(
+			s,
+			getIdentityForRequestSpecificToNode(s, action.Identity, nodeIndex),
```

Comment:
question: Why do we need a request token for a direct to node (non-network) call?

Reply:
I don't think this will ever be the case. Tokens are for external requests. A start command is actioned by a local user (I consider an ssh terminal to be local here). I would suggest passing the type of identity that is required for that command.